### ISSUE TITLE
[Accessibility][Parsing - Notification]: Ensures all ARIA attributes have valid values.

### ISSUE THREAD

**Issue by ShellySoni** [microsoft/BotFramework-Composer#2124]
**User Experience:**
This issue should be resolved for good coding practice if all ARIA attributes have valid values.

**Test Environment:**
Environment: Chromium Edge (Anaheim) + Narrator
Build Version: 2004 (OS Build 19564.1005)​​
Microsoft Edge Version: 82.0.425.0 (Official build) canary (64-bit)​
Test URL: http://localhost:3000/home

**Repro Steps:**

1. Open URL: http://localhost:3000/home
2. "Bot Framework Composer" homepage will gets opened.
3. Select any of the previously created bot from "Recent Bots" or create one from the available Bots under 'Examples' and activate it.
4. Select 'Notification' from left navigation.
5. Run K4W tool on the page and Observe the issue

**Actual Result:**
All ARIA attributes i.e. 'Type' and 'Location' table columns does not have valid values.

This accessibility issue was found using Accessibility Insights for Web(keros), a tool that helps debug and find accessibility issues earlier. Get more information & download this tool at http://aka.ms/Keros.

**Expected Result:**
Ensures all ARIA attributes  i.e. 'Type' and 'Location' table columns have valid values.

**Note:**
Same issue observed for 'Design Flow' pane.
Same issue observe throughout the website for every property pane. {Attached some of screenshot below, please look the issue throughout the application}

**MAS Reference:**
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={2e2a1540-aaf7-48a0-a53e-8f10cf309999}

**Attachment:**
[MAS4.1.1_Notification_table Headers_K4w.zip](https://github.com/microsoft/BotFramework-Composer/files/4266403/MAS4.1.1_Notification_table.Headers_K4w.zip)
![ensure all aria attributes have valid values](https://user-images.githubusercontent.com/60919065/75653277-4bc69c80-5c83-11ea-9cc5-e69696046fd9.png)

**Comment by corinagum** [comment]
![MAS4 1 1_Notification__Table Headers_K4W](https://user-images.githubusercontent.com/14900841/75576482-16b51280-5a15-11ea-93e8-3f59b4707c32.png)
![MAS4 1 1_Notification_table Headers_K4w](https://user-images.githubusercontent.com/14900841/75576490-174da900-5a15-11ea-9b23-2e24bcd836d4.png)

**Comment by ashish315** [comment]
As checked the issue is resolved. So HCL is closing the bug
